pages/[type]/[id]/moderation.vue

```diff
@@ -0,0 +1,216 @@
+        Project reviews typically take 24 to 48 hours and they will leave a message below if they
+        have any questions or concerns for you. If your review has taken more than 48 hours, check
+        our Discord or social media for moderation delays.
+      </p>
+      <template v-else-if="isRejected(project)">
+        <p>
+          Your project does not currently meet Modrinth's
+          <nuxt-link to="/legal/rules" class="text-link" target="_blank">content rules</nuxt-link>
+          and the moderators have requested you make changes before it can be approved. Read the
+          messages from the moderators below and address their comments before resubmitting.
+        </p>
+        <p class="warning">
+          Repeated submissions without addressing the moderators' comments may result in an account
+          suspension.
+        </p>
+      </template>
+      <h3>Current visibility</h3>
+      <ul class="visibility-info">
+        <li v-if="isListed(project)">
+          <CheckIcon class="good" />
+          Listed in search results
+        </li>
+        <li v-else>
+          <ExitIcon class="bad" />
+          Not listed in search results
+        </li>
+        <li v-if="isListed(project)">
+          <CheckIcon class="good" />
+          Listed on the profiles of members
+        </li>
+        <li v-else>
+          <ExitIcon class="bad" />
+          Not listed on the profiles of members
+        </li>
+        <li v-if="isPrivate(project)">
+          <ExitIcon class="bad" />
+          Not accessible with a direct link
+        </li>
+        <li v-else>
+          <CheckIcon class="good" />
+          Accessible with a direct link
+        </li>
+      </ul>
+    </section>
+    <section id="messages" class="universal-card">
+      <h2>Messages</h2>
+      <p>
+        This is a private conversation thread with the Modrinth moderators. They will message you
+        for issues concerning your project on Modrinth, and you are welcome to message them about
+        things concerning your project.
+      </p>
+      <template v-if="thread">
+        <ConversationThread
+          v-if="thread"
+          :thread="thread"
+          :update-thread="(newThread) => (thread = newThread)"
+          :project="project"
+          :set-status="setStatus"
+          :current-member="currentMember"
+        />
+      </template>
+      <div v-else class="known-errors unavailable-error">
+        <WarningIcon /> Messages are currently not available on this project.
+        <span v-if="config.public.apiBaseUrl.startsWith('https://staging-api.modrinth.com')">
+          This is expected on staging because the database has not been fully migrated, please make
+          a new project to try message threads.
+        </span>
+        <span v-else>
+          If you're seeing this, please report this as an issue including your project's ID:
+          <CopyCode :text="project.id" />
+        </span>
+      </div>
```

Comment:
you can remove this, as threads are mandatory for reports + projects now

Reply:
seems better to have an error message than not, no? I could just simplify it though and remove the first part

Reply:
the message would never show up, because projects without threads are invalid in the DB and won't be returned. so the code can be removed